src/java.management.rmi/share/classes/javax/management/remote/rmi/RMIConnection.java

```diff
@@ -123,9 +123,7 @@ public interface RMIConnection extends Closeable, Remote {
      *
      * @param className The class name of the MBean to be instantiated.
      * @param name The object name of the MBean. May be null.
-     * @param delegationSubject The <code>Subject</code> containing the
-     * delegation principals or <code>null</code> if the authentication
-     * principal is used instead.
+     * @param delegationSubject No longer supported and should be null.
```

Comment:
I think this interface will require all methods that take a delegation subject to specify UOE then the subject is not null.